tmt/steps/provision/testcloud.py

```diff
@@ -1384,3 +1399,66 @@ def clean_images(cls, clean: 'tmt.base.Clean', dry: bool, workdir_root: Path) ->
                     clean.fail(f"Failed to remove '{image}'.", shift=2)
                     successful = False
         return successful
+
+
+@container
+class ConsoleLog(tmt.steps.provision.GuestLog):
+    # Linked guest
+    guest: GuestTestcloud
```

Comment:
I'd say we should move this one into the parent class, it feels like a useful object to be able to reach and use from the guest log code.

Reply:
Good point, moved in 953e788e.